src/murmur/Messages.cpp

```diff
@@ -1403,12 +1406,19 @@ void Server::msgTextMessage(ServerUser *uSource, MumbleProto::TextMessage &msg)
 			return;
 		}
 
+		// Users directly in that channel
 		foreach (User *p, c->qlUsers)
 			users.insert(static_cast< ServerUser * >(p));
 
+		// Users only listening in that channel
+		foreach (unsigned int session, ChannelListener::getListenersForChannel(c))
+			users.insert(qhUsers.value(session));
```

Comment:
```suggestion
		foreach (unsigned int session, ChannelListener::getListenersForChannel(c)) {
			users.insert(qhUsers.value(session));
		}
```
I know a lot of other code isn't using braces as well, but I am trying to introduce them everywhere when working on the respective parts :point_up:

Reply:
Funny, because I prefer braces in that cases, too. I just adapted to the existing code style. I will change that :+1: